Add a receiver query argument to override default receiver

kchugalinskiy opened the pull request:
We need a way to make phone call to a different people and it depends on prometheus event. I see no motivation to run a separated promtotwilio for each person we should call.

This pull request makes `RECEIVER` env var optional, adding a `receiver` query parameter. If none is set, BadRequest should be returned.

Swatto commented:
Hi @kchugalinskiy. Thanks for your proposal and it would be a great addition.

Before accepting this PR, could you:

- Run `gofmt` to respect the code formating
- Add and explain this feature in the `README.md` and mention the fact that the env var `RECEIVER` becomes optional

kchugalinskiy commented:
Sounds reasonable. I've fulfilled your requirements. Please take a look. Also I've added one error check `_, err := jsonparser.ArrayEach() ... log.Warn(err)` and removed false comparation here: `!ctx.IsPost()` instead of  `ctx.IsPost() == false`

Swatto commented:
👍
